app/models/repository.rb

```diff
@@ -1,5 +1,3 @@
-require 'securerandom'
```

Comment:
This was already removed in master https://github.com/SUSE/rmt/blob/master/app/models/repository.rb ^_^